### DISCUSSION TITLE
월드 내에서 플레이어끼리의 모습이 제대로 전송이 안됩니다

### DISCUSSION THREAD

**Discussion by JeongYire** [naverz/zepeto-studio-kor#280] [5. Q&A]
멀티플레이 샘플을 적용시킨후, Spawn Info 외에 일체 바꾸지 않고 빌드해서 배포했습니다만.
멀티플레이까지는 가능한데 다른 사람은 분명 가만히있는데 제가 보고있는 화면에서는 벽에 붙어서 달려가고있는것처럼 보이고
서로간의 정보공유가 제대로 되지않는것처럼 보입니다. 이 경우는 제페토 내 서버의 문제인지요? 
2022-01-27 16:50 기준입니다

**Comment by x4cf** [comment]
샘플은 정말 최소한만 작동하는 샘플입니다. 위치가 멀어지면 서로의 위치를 맞추거나 서로의 모습을 더 자연스럽게 보이도록 동기화 하는건 개발자가 해야할 내용입니다.
서버에는 아무 문제 없는것으로 보입니다

**Reply by JeongYire** [reply]
그럼 현재 제공되는 멀티플레이샘플에서 서로 위치가 전달되지못해 벽에 붙어 계속 달리고있는다던지, 점프해도 점프한것처럼 안보인다던지의 문제는 제페토개발자분들이 샘플로서만 보여주기위해 아주 최소한으로만 만들었다고 생각하면 되나요? 이런 이슈에대해선 그럼 어디부터 출발해야될지 모르겠습니다 . 멀티플레이 샘플에 작성된 코드로만 보면 더이상 뭘 추가해야될지 제 실력으로는 감이 안잡히네요

**Reply by x4cf** [reply]
서버 로직 프로그래밍은 어렵습니다. 
탬플릿이 아닌 샘플은 그저 샘플이며, 샘플만으로 비 전공자 개인이 도전하기에는 무리가 있죠.
질문이 정리가 된다면 깃에 올려보기도 하고 온라인 커뮤니티에서 힌트를 얻는것도 좋은 방법입니다.

https://open.kakao.com/o/sHLhciId
제페토 월드 개발자들의 톡방에 오셔서 가볍게 질문을 해주시는것도 좋아요 :)
